src/Addon/Event/AddonStartedEvent.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// Copyright (C) 2010-2024, the Friendica project
+// SPDX-FileCopyrightText: 2010-2024 the Friendica project
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+declare(strict_types=1);
+
+namespace Friendica\Addon\Event;
+
+/**
+ * Start an addon.
+ */
+final class AddonStartedEvent
```

Comment:
This is a finer language point of course, but I feel like it's time to pick. Either we should make all event class names include a past participle like `AddonStartedEvent`, or we should make all event class names include an infinitive like `AddonInstallEvent`, but I don't think we should mix. I'd prefer infinitive as they convey the same information in a smaller package.
```suggestion
final class AddonStartEvent
```

Reply:
Agree. Thank you for pointing this out. Fixed.